shared.nix

```diff
@@ -0,0 +1,90 @@
+let sources = import ./nix/sources.nix; in
+{ haskellNix ? import sources.haskell-nix { sourcesOverride = sources; }
+, haskellNixOverlays ? (pkgsNew: pkgsOld: { haskell-nix = pkgsOld.haskell-nix // { hackageSrc = sources.hackage-nix; }; })
+, nixpkgsSrc ? haskellNix.sources.nixpkgs-2003
+, haskellCompiler ? "ghc883"
+, shellBuildInputs ? []
+, ...
+}:
+let
+  libtorchSrc = pkgs:
+    let src = pkgs.fetchFromGitHub {
+      owner = "stites";
+      repo = "pytorch-world";
+      rev = "6dc929a791918fff065bb40cbc3db8a62beb2a30";
+      sha256 = "140a2l1l1qnf7v2s1lblrr02mc0knsqpi06f25xj3qchpawbjd4c";
+    };
```

Comment:
I've never actually used niv, so I'm not completely sure about this, but maybe this should be managed with `niv` as well?

Reply:
you are absolutely right! good catch, this was just copy-pasted from the old derivations :D

Reply:
I moved this to `sources.json` :)